src/javascript/app/Stores/Modules/Trading/trade-store.js

```diff
@@ -108,6 +108,10 @@ export default class TradeStore extends BaseStore {
     @observable proposal_info = {};
     @observable purchase_info = {};
 
+    // Loading Status
+    @observable is_slow_loading = false;
```

Comment:
can we put `is_slow_loading ` and  ` slow_loading_status` in `ui-store`? Afaik trade-store is for trade parameters.